core/include/spi.h

```diff
@@ -30,30 +30,18 @@
 
 #include <types_ext.h>
 
```

Comment:
Regarding magic numbers and macros, yes i would personally prefer macros as they explicitly state the supported values in the source code. I agree execution if not affected since allowed values are verified.